csfieldguide/static/interactives/picture-memory/js/picture-memory.js

```diff
@@ -0,0 +1,158 @@
+var items = [
+  gettext("backpack"),
+  gettext("banana"),
+  gettext("bike"),
+  gettext("book"),
+  gettext("cactus"),
+  gettext("calculator"),
+  gettext("camera"),
+  gettext("chair"),
+  gettext("cup"),
```

Comment:
I still think 'mug' every time I look at the picture of this - it's a tall, opaque, and mostly straight vessel with steam. I'd associate a cup as holding a cool liquid or a teacup as being short and much wider at the brim